misc/inmanta-workon-register.sh

```diff
@@ -144,6 +144,13 @@ function __inmanta_workon_list {
 
 
 function __inmanta_workon_activate {
+    # Check user
+    current_user = $(whoami)
```

Comment:
Here I hesitated with `$USER`, not sure if one is better than the other in this context.

Reply:
I think `whoami` is more portable.